requirements.txt

```diff
@@ -1,95 +1,104 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.10
```

Comment:
Please could you generate this with Python 3.9 to avoid getting library versions that are too new?

Reply:
Actually never mind, we aren't running Debian...